apps/fluent-tester/src/E2E/Avatar/specs/Avatar.spec.win.ts

```diff
@@ -22,3 +32,25 @@ describe('Avatar Testing Initialization', function () {
     expect(AvatarPageObject.didAssertPopup()).toBeFalsy(AvatarPageObject.ERRORMESSAGE_ASSERT);
   });
 });
+
+describe('Avatar Accessibility Testing', () => {
+  beforeEach(() => {
+    AvatarPageObject.scrollToTestElement();
+    AvatarPageObject.waitForPrimaryElementDisplayed(PAGE_TIMEOUT);
+  });
+  it('Validate accessibilityLabel', () => {
+    expect(AvatarPageObject.getPrimaryComponentAttribute(ACCESSIBILITY_LABEL_ATTR)).toEqual(JSAVATAR_ACCESSIBILITY_LABEL);
```

Comment:
You should add a check for didAssertPopup after each attribute check (similar to how other component specs do).